src/main/java/io/confluent/connect/elasticsearch/Mapping.java

```diff
@@ -16,12 +16,11 @@
 
 package io.confluent.connect.elasticsearch;
 
-import com.google.gson.JsonObject;
-
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.JsonNodeFactory;
 import com.fasterxml.jackson.databind.node.ObjectNode;
-
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
Although guava is generally available via other dependencies we import, we avoid importing it explicitly as a dependency in connectors. We might change this in the future, but that's the current status. I'm not feeling very strong about this, but probably a decision to use it should be based on a more essential use case than this annotation.

Reply:
Ok, I can replace with a comment.